README.md

```diff
@@ -119,23 +119,57 @@ cargo build -p millau-bridge-node
 cargo build -p substrate-relay
 ```
 
-### Running
+### Running dev network
+
+We will only launch a dev network to demonstrate the unidirectional relay from millau to rialto here
+
+#### Running from local script
 
 To run a simple dev network you'll can use the scripts located in
 [the `deployments/local-scripts` folder](./deployments/local-scripts). Since the relayer connects to
-both Substrate chains it must be run last.
+both Substrate chains it must be run last. 
 
 ```bash
-# In `parity-bridges-common` folder
+
+# run rialto node
 ./deployments/local-scripts/run-rialto-node.sh
+
+# run millau node
 ./deployments/local-scripts/run-millau-node.sh
+
+# init bridge & relay headers from millau to rialto
 ./deployments/local-scripts/relay-millau-to-rialto.sh
+
+# relay messages from millau to rialto
+./deployments/local-scripts/relay-messages-millau-to-rialto.sh
 ```
 
-At this point you should see the relayer submitting headers from the Millau Substrate chain to the
-Rialto Substrate chain.
+At this point you should see the relayer submitting headers/messages from the Millau Substrate chain to the
+Rialto Substrate chain as following:
 
-### Local Docker Setup
+``` from relay headers
```

Comment:
```suggestion

# Header Relayer Logs
```

GitHub is rendering this a bit strange, but leave comment instead of annotating the ticks.